benchmarks/fplx_evaluation.py

```diff
@@ -86,7 +96,7 @@ def process_fplx_groundings(df):
             if k == 'PUBCHEM':
                 chebi_id = chebi_client.get_chebi_id_from_pubchem(v)
                 if chebi_id:
-                    grounding['db_refs']['CHEBI'] = 'CHEBI:%s' % chebi_id
```

Comment:
This makes me very happy 😄

Reply:
Lol yes, it makes me cringe that `chebi_client.get_chebi_id_from_pubchem` used to return a CHEBI ID without the prefix until I changed it.

Reply:
Oh... then I'm happy for the wrong reason. I thought you were finally coming to the dark side with non-redundant identifiers